redfish-core/lib/systems.hpp

```diff
@@ -3005,10 +3233,15 @@ inline void handleComputerSystemCollectionGet(
     ifaceArray = nlohmann::json::array();
     if constexpr (bmcwebEnableMultiHost)
     {
-        asyncResp->res.jsonValue["Members@odata.count"] = 0;
-        // Option currently returns no systems.  TBD
+        constexpr std::array<std::string_view, 1> interfaces{
+            /*xyz.openbmc_project.Inventory.Item.System*/
```

Comment:
inline comments

Reply:
Same as in earlier comment, after performing testing and verifying which way works the best, the comment will be thrown away.